### PR TITLE
Resin sync features documentation update

### PR DESCRIPTION
**Pull request by lekkas**

### CODE REVIEW

File: pages/tools/cli.md [hunk 1]
```diff
@@ -582,65 +582,82 @@ continuously stream output
...
-## sync [source]
+## sync [uuid]
 
 WARNING: If you're running Windows, this command only supports `cmd.exe`.
 
 Use this command to sync your local changes to a certain device on the fly.
 
-The `source` argument can be either a device uuid or an application name.
+After every 'resin sync' the updated settings will be saved in
+'<source>/.resin-sync.yml' and will be used in later invocations. You can
```
Comment (shaunmulligan) [inline pages/tools/cli.md]: @lekkas so the file changed to a dotfile right? will it recognise older versions that don't use the dotfile but rather just have `resin-sync.yml`

Comment (lekkas) [inline pages/tools/cli.md]: right, we changed it to a dotfile to follow the convention of having dotfiles for third-party app config files (e.g. `.gitignore`, `.*rc files etc) and it doesn't parse `resin-sync.yml` files - only `.resin-sync.yml`

Comment (lekkas) [inline pages/tools/cli.md]: @shaunmulligan what if resin sync detected `resin-sync.yml` and converted it to `.resin-sync.yml` after prompting the user accordingly? The new `.resin-sync.yml` is compatible with the older format although some options are ignored:

- `progress` and `verbose`, being debugging options are no longer persisted 
- `source` is ignored - `.resin-sync.yml` is now expected to always be in the root of the project directory to be synced

Comment (shaunmulligan) [inline pages/tools/cli.md]: @lekkas yeah would be cool if it converted if for the user after a prompt 👍

Comment (lekkas) [inline pages/tools/cli.md]: on it - https://github.com/resin-io-modules/resin-sync/issues/40

Comment (lekkas) [inline pages/tools/cli.md]: @shaunmulligan PR'ed - does this user prompt look good ? https://github.com/resin-io-modules/resin-sync/pull/41#issue-173503407

**Comment by shaunmulligan** [comment]
@lekkas sorry for the late reply. Yeah that message looks great :+1:

**Comment by lekkas** [comment]
@shaunmulligan no worries, PR merged!

**Pull request merged**